/* captura */
try {
    let error = 5/0;
    console.log("no ejecutada");
} catch(exception){
    console.log("excepci贸n capturada");
} finally {
    console.log("finally");
}
            
/* captura y delegaci贸n */
try {
    try {
        let error = 5/0;
        console.log("no ejecutada");
    } catch(exception){
        console.log("excepci贸n capturada");
        throw excpetion;
    }
} catch(exception){
    console.log("excepcion delegada");
    console.log(exception);
}
            
/* elevaci贸n */
try {
    throw "exception elevada";
    console.log("no ejecutada");
} catch (expection){
    console.log(expection);
}
